plugins/processors/awsapplicationsignals/internal/resolver/kubernetes.go

```diff
@@ -501,13 +390,67 @@ func (m *ServiceToWorkloadMapper) Start(stopCh chan struct{}) {
 			case <-stopCh:
 				return
 			case <-time.After(time.Minute + 30*time.Second):
-				m.MapServiceToWorkload()
+				m.mapServiceToWorkload()
 				m.logger.Debug("Map service to workload at:", zap.Time("time", time.Now()))
 			}
 		}
 	}()
 }
 
+func filterPodIPFields(obj interface{}) (interface{}, error) {
+	if pod, ok := obj.(*corev1.Pod); ok {
+		pod.Annotations = nil
+		pod.Finalizers = nil
+		pod.ManagedFields = nil
+
+		pod.Spec.Volumes = nil
+		pod.Spec.InitContainers = nil
+		pod.Spec.EphemeralContainers = nil
+		pod.Spec.ImagePullSecrets = nil
+		pod.Spec.HostAliases = nil
+		pod.Spec.SchedulingGates = nil
+		pod.Spec.ResourceClaims = nil
+		pod.Spec.Tolerations = nil
+		pod.Spec.Affinity = nil
+
+		pod.Status.InitContainerStatuses = nil
+		pod.Status.ContainerStatuses = nil
```

Comment:
please add a comment to indicate which fields will not be reset that will be used by the processor

Reply:
added.